emData/download.sh

```diff
@@ -205,40 +169,13 @@ grep -v TP_ ../LUTsSplit/processingmodules.dat | grep -v VMR_ | grep -v IR_ > fp
 sed -i 's/VMStubMERouter/VMSMERouter/g' fpga2_processingmodules.dat
 sed -i 's/VMStubMERouter/VMSMERouter/g' fpga1_processingmodules.dat
 
-./makeReducedConfig.py --no-graph -t "TP" -s "C" -o "reducedcm_"
-cp -fv ../LUTsCM2/wires.dat ../LUTsCM2/memorymodules.dat ../LUTsCM2/processingmodules.dat ./
-mv wires.dat reducedcm2_wires.dat
-mv memorymodules.dat reducedcm2_memorymodules.dat
-mv processingmodules.dat reducedcm2_processingmodules.dat
-### Reduced Combined IRtoTB
 echo "Reduced CM"
-./generator_hdl.py ../../ --no_graph --mut IR -u 0 -d 4 -w reducedcm_wires.dat -p reducedcm_processingmodules.dat -m reducedcm_memorymodules.dat -de 1
-./generator_hdl.py ../../ --no_graph --mut IR -u 0 -d 4 -w reducedcm_wires.dat -p reducedcm_processingmodules.dat -m reducedcm_memorymodules.dat -de 1 -x
-mkdir -p ../../IntegrationTests/ReducedCombinedConfig/{hdl,tb}
-mv -fv memUtil_pkg.vhd SectorProcessor.vhd SectorProcessorFull.vhd ../../IntegrationTests/ReducedCombinedConfig/hdl/
-mv -fv tb_tf_top.vhd ../../IntegrationTests/ReducedCombinedConfig/tb/
-### Reduced Combined 2 IRtoTB
-echo "Reduced CM2"
-./generator_hdl.py ../../ --no_graph --mut IR -u 0 -d 4 -w reducedcm2_wires.dat -p reducedcm2_processingmodules.dat -m reducedcm2_memorymodules.dat
-./generator_hdl.py ../../ --no_graph --mut IR -u 0 -d 4 -w reducedcm2_wires.dat -p reducedcm2_processingmodules.dat -m reducedcm2_memorymodules.dat -x
-mkdir -p ../../IntegrationTests/ReducedCombinedConfig2/{hdl,tb}
-mv -fv memUtil_pkg.vhd SectorProcessor.vhd SectorProcessorFull.vhd ../../IntegrationTests/ReducedCombinedConfig2/hdl/
-mv -fv tb_tf_top.vhd ../../IntegrationTests/ReducedCombinedConfig2/tb/
-### Reduced Combined 2 IRtoTB
-echo "CM Barrel"
-./generator_hdl.py ../../ --no_graph --mut IR -u 0 -d 4 -w cmbarrel_wires.dat -p cmbarrel_processingmodules.dat -m cmbarrel_memorymodules.dat -de 1
-./generator_hdl.py ../../ --no_graph --mut IR -u 0 -d 4 -w cmbarrel_wires.dat -p cmbarrel_processingmodules.dat -m cmbarrel_memorymodules.dat -de 1 -x
-mkdir -p ../../IntegrationTests/CombinedBarrelConfig/{hdl,tb}
-mv -fv memUtil_pkg.vhd SectorProcessor.vhd SectorProcessorFull.vhd ../../IntegrationTests/CombinedBarrelConfig/hdl/
-mv -fv tb_tf_top.vhd ../../IntegrationTests/CombinedBarrelConfig/tb/
-### Combined IRtoTP
-echo "CombinedIRtoTP"
-./generator_hdl.py ../../ --no_graph --mut IR -u 0 -d 2 -w fpga1_wires.dat -p fpga1_processingmodules.dat -m fpga1_memorymodules.dat -de 1 -sp 1
-./generator_hdl.py ../../ --no_graph --mut IR -u 0 -d 2 -w fpga1_wires.dat -p fpga1_processingmodules.dat -m fpga1_memorymodules.dat -de 1 -x -sp 1
-mkdir -p ../../IntegrationTests/CombinedConfig_FPGA1/{hdl,tb}
-mv -fv memUtil_pkg.vhd SectorProcessor.vhd SectorProcessorFull.vhd ../../IntegrationTests/CombinedConfig_FPGA1/hdl/
-mv -fv tb_tf_top.vhd ../../IntegrationTests/CombinedConfig_FPGA1/tb/
```

Comment:
@aryd The generation of the VHDL files for FPGA1 has been removed here. Is this intentional?

Reply:
Oops - not sure how this happened - I'm adding this code back.